helm/ako/crds/nsxalbinfra.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: nsxalbinfrasettings.ako.vmware.com
+spec:
+  conversion:
+    strategy: None
+  group: ako.vmware.com
+  names:
+    kind: NsxAlbInfraSetting
+    listKind: NsxAlbInfraSettingList
+    plural: nsxalbinfrasettings
+    shortNames:
+    - nsxalbinfrasettings
+    - nas
+    singular: nsxalbinfrasetting
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: NSX ALB Settings is used to select specific Avi controller infra attributes.
+        properties:
+          spec:
+            properties:
+              network:
+                properties:
+                  name:
+                    type: string
+                properties:
+                  rhi:
+                    type: string
+                type: object
+                required:
+                - name
+              seGroup:
+                properties:
+                  name:
+                    type: string
+                type: object
+            type: object
+          status:
+            properties:
+              error:
+                type: string
+              status:
+                type: string
+            type: object
+        required:
```

Comment:
instead of making it required, do we need to make it optional so that maybe we can enforce a default setting.

Reply:
As discussed, at least 1 required field is needed to not let users create an empty CRD.